data/src/main/java/org/akvo/flow/data/datasource/files/BitmapHelper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017-2018 Stichting Akvo (Akvo Foundation)
+ * Copyright (C) 2018 Stichting Akvo (Akvo Foundation)
```

Comment:
So this is just very odd, I moved some code from ImageDataSource to BitmapHelper and also moved them to another package and somehow github got it all mixed up.